profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -205,6 +205,15 @@ function _tmgmt_poetry_form_load_translator_settings(&$form_state, &$form, $job)
     $settings = [];
   }
 
+  // NEPT-62: Overwriting the 'delai' field properties.
+  if (isset($settings['delai'])) {
```

Comment:
Actually we want to override the label. The date can be changed by DGT, and the expected delivery date won't be the requested delivery date if that is changed.
So if it was set, we change the label to differenciate this with the date that will show in the jobs